functions/java-jre.bash

```diff
@@ -232,3 +239,28 @@ java_zulu_install_crypto_extension(){
   rm -rf "${policyTempLocation}"
   return 0
 }
+
+adoptopenjdk_install_apt(){
+  local adoptKey="/tmp/adoptopenjdk.asc"
+  if ! dpkg -s 'adoptopenjdk-8-hotspot-jre' >/dev/null 2>&1; then # Check if already is installed
+    echo -n "$(timestamp) [openHABian] Adding AdoptOpenJDK keys to apt... "
+    wget -qO "$adoptKey" https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public
+    if cond_redirect apt-key add "$adoptKey"; then echo "OK"; else echo "FAILED (keyserver)"; exit 1; fi
```

Comment:
if key fetch fails this leaves an empty file so remove it when fetch fails, too

Reply:
BTW also fix that in https://github.com/ecdye/openhabian/blob/AdoptOpenJDK/functions/java-jre.bash#L133

Reply:
Fixed